### PR TITLE
Update spec.md

### PR DESCRIPTION
**Pull request by decanus**

### CODE REVIEW

File: specs/spec.md [hunk 1]
```diff
@@ -99,7 +100,7 @@ The `body` field contains an [SSZ](https://github.com/ethereum/eth2.0-specs/blob
 Below you will find an example `RPC` message.
 
 ```
-EWP 0.2 RPC 0 65
+EWP30650
```
Comment (atoulme) [inline specs/spec.md]: Is 3 encoded as 4 bytes?

Comment (decanus) [reply specs/spec.md]: no, 1 byte. CHeck the spec

**Review by jrhea — APPROVED** [review]
lgtm

File: specs/spec.md (hunk 1 above)
Comment (jrhea) [reply specs/spec.md]: this is outdated. dean fixed it

**Pull request merged**